### PR TITLE
add multi command line

### PR DESCRIPTION
**Pull request by n2oneProgrammer**

### CODE REVIEW

File: static/css/main.css [hunk 1]
```diff
@@ -152,6 +152,37 @@
     max-height: 100%
 }
 
+.aside__multiline-section {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
```
Comment (mwawrzen) [inline static/css/main.css]: useless semicolon

File: static/css/main.css [hunk 2]
```diff
@@ -152,6 +152,37 @@
...
+}
+
+.aside__multiline {
+    display: none;
+    position: relative
+}
+
+.multiline-button {
+    width: 100% !important;
+    height: 42px;
```
Comment (mwawrzen) [inline static/css/main.css]: useless semicolon

File: static/js/ScreenManager.js [hunk 3]
```diff
@@ -60,7 +61,9 @@ class ScreenManager {
             this.terminalButtons[name.replaceAll('-', '_')] = button;
         });
         this.commandLine = document.querySelector("#command_line");
+        this.multiCommandLine = document.querySelector(".aside__multiline-multiline");
         this.commandLineButton = document.querySelector(".aside__input-confirm");
+
```
Comment (mwawrzen) [inline static/js/ScreenManager.js]: useless empty line

**Review by mwawrzen — APPROVED** [review]
LGTM

**Review by jaanonim — APPROVED** [review]
LGTM

**Pull request merged**